be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -75,8 +75,9 @@ Status KafkaDataConsumer::init(StreamLoadContext* ctx) {
     // TODO: set it larger than 0 after we set rd_kafka_conf_set_stats_cb()
     RETURN_IF_ERROR(set_conf("statistics.interval.ms", "0"));
     RETURN_IF_ERROR(set_conf("auto.offset.reset", "error"));
-    RETURN_IF_ERROR(set_conf("api.version.request", "true"));
+    RETURN_IF_ERROR(set_conf("api.version.request", config::kafka_api_version_request));
```

Comment:
Is It correct when api.version.request=true and broker.version.fallback=0.9 for new version of libdfkafka

Reply:
![image](https://user-images.githubusercontent.com/55968745/148339562-813bbe5c-2aea-40d1-8319-b486231501b6.png)
I will set broker.version.fallback=0.10.0 for default

Reply:
Great ~